src/main/scala/ru/org/codingteam/horta/plugins/pet/Extractors.scala

```diff
@@ -0,0 +1,19 @@
+package ru.org.codingteam.horta.plugins.pet
+
+object DyingPet {
+
+  def unapply(pet: PetData) = pet.health <= 0 || pet.satiation <= 0
```

Comment:
`Boolean` is possible too. Its purpose is obvious, see more at http://daily-scala.blogspot.ru/2009/10/boolean-extractors.html